|gse   |title                                                             |pubmed_id                                             |summary                                |overall_design                                                    |gpl     |batch | n genes| n samples| channel_count|
|:--------|:------------------------------------------------------------------------------------------------------------------------------------------|:-----------------------------------------------------|:-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|:---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|:-------|:-------------------------|-------:|---------:|-------------:|
|GSE1379  |breast cancer / tamoxifen monotherapy (whole tissue tumor biopsies)                                                                        |[15193263](https://pubmed.ncbi.nlm.nih.gov/15193263/) |dataset of 60 patients with ER-positive primary breast cancer and treated with tamoxifen monotherapy for 5 years. Data were generated from whole tissue sections of breast cancers.;	Sample_keyword: breast cancer, tamoxifen, recurrence;	Keywords: other |NA                                                                |GPL1223 |NA                        |   14801|        60|             2|
|GSE2034  |Breast cancer relapse free survival |[15721472](https://pubmed.ncbi.nlm.nih.gov/15721472/) |This series represents 180 lymph-node negative relapse free patients and 106 lymph-node negate patients that developed a distant metastasis.;	Please see attached patient clinical parameters sheet for more information.;	Keywords: other       |NA  |GPL96   |NA                        |   12722|       286|             1|
|GSE9893  |A gene expression signature predicting the recurrence of tamoxifen-treated primary breast cancer.                                          |[18347175](https://pubmed.ncbi.nlm.nih.gov/18347175/) |A 36-gene classifier was constructed through expression profiling of 132 tumors  from tamoxifen-treated patients using 70-mer oligonucleotide microarrays. The robustness of the signature was demonstrated using expression data from 83 independent tumors. The 36-gene signature was (i) more efficient to predict disease-free survival than the traditional histo-pathological prognostic factors, (ii) as effective as the Nottingham Prognostic Index or the "Adjuvant!" software, and (iii) the only independent prognostic factor. Comparison with several already published signatures demonstrated that the 36-gene signature was among the best to classify tumors.;	Keywords: Gene expression profiling; supervised analysis; molecular signature predictive of recurrence; univariate and multivariate analysis in relation to disease-free survival.                                                          |In the study presented here, a cohort of 132 primary tumors from tamoxifen-treated patients followed up more than 5-years, was used to acquire expression profiles at the whole genome level by 70-mer oligonucleotide microarrays containing 22,680 probes (which represent 21,329 human specific genes). Supervised predictive analysis of microarrays allowed construction of a 36-gene molecular classifier whose robustness was assessed by using gene expression data from 83 independent tumors (including 23 tumors from our microarray platform and 60 tumors from the study by Ma et al., Cancer cell 2004; 5:607-616). |GPL5049 |NA                        |   13154|       155|             1|
|GSE12093 |The 76-gene Signature Defines High-Risk Patients that Benefit from Adjuvant Tamoxifen Therapy                                              |[18821012](https://pubmed.ncbi.nlm.nih.gov/18821012/) |Classification of tamixifen-treated breast cancer patients into high and low risk groups using the 76-gene signature                                                          |136 breast cancer samples that were treated with tamoxifen were classified using the 76-gene signature                                                          |GPL96   |NA                        |   12722|       136|             1|
|GSE16391 |GGI: a potential predictor of relapse for endocrine-treated breast cancer patients in the BIG 1-98 trial                                   |[19573224](https://pubmed.ncbi.nlm.nih.gov/19573224/) |Background: We have previously shown that the Gene expression Grade Index (GGI) was able to identify two subtypes of estrogen receptor (ER)-positive tumors that were associated with statistically distinct clinical outcomes in both untreated and tamoxifen-treated patients. Here, we aim to investigate the ability of the GGI to predict relapses in postmenopausal women who were treated with tamoxifen (T) or letrozole (L) within the BIG 1-98 trial.;	Methods: We generated gene expression profiles (Affymetrix) and computed the GGI for a matched, case-control sample of patients enrolled in the BIG 1-98 trial from the two hospitals where frozen samples were available. All relapses (cases) were identified from patients randomized to receive monotherapy or from the switching treatment arms for whom relapse occurred before the switch.  Each case was randomly matched with four controls based upon nodal status and treatment (T or L).  The prognostic value of GGI was assessed as a continuous predictor and divided at the median.  Predictive accuracy of GGI was estimated using time-dependent area under the curve (AUC) of the ROC curves. ;	Results:  Frozen samples were analyzable for 48 patients (10 cases and 38 controls).  Seven of the 10 cases had been assigned to receive L. Cases and controls were comparable with respect to menopausal and nodal status, local and chemotherapy, and HER2 positivity. Cases were slightly older than controls and had a larger proportion of large, poorly differentiated ER+/PgR- tumors.  The GGI was significantly and linearly related to risk of relapse: each 10-unit increase in GGI resulted in an increase of approximately 11% in the hazard rate (p=0.02).  Within the subgroups of patients with node-positive disease or who were treated with L, the hazard of relapse was significantly greater for patients with GGI at or above the median. AUC reached a maximum of 78% at 27 months.;	Conclusions: This analysis supports the GGI as a good predictor of relapse for ER-positive patients, even among patients who receive L.  Validation of these results, in a larger series from BIG 1-98, is planned using the simplified GGI represented by a smaller set of genes and tested by qRT-PCR on paraffin-embedded tissues. |Test whether the Gene expression Grade Index (GGI) is a potential predictor of relapse for endocrine-treated breast cancer patients in the BIG 1-98 trial.;	;	55 microarray experiments from primary breast tumors of endocrine-treated patients.;	;	No replicate, no reference sample. |GPL570  |NA                        |   18260|        48|             1|
|GSE16446 |Multifactorial Approach to Predicting Resistance to Anthracyclines                                                                         |[21422418](https://pubmed.ncbi.nlm.nih.gov/21422418/) |PURPOSE: Validated biomarkers predictive of response/resistance to anthracyclines in breast cancer are currently lacking. The neoadjuvant TOP trial, in which patients with estrogen receptor (ER)-negative tumors were treated with anthracycline (epirubicin) monotherapy, was specifically designed to evaluate the predictive value of topoisomerase IIα (TOP2A) and to develop a gene expression signature to identify those patients who do not benefit from anthracyclines.;	;	METHODS: The TOP trial included 149 patients, of which 141 were evaluable for response prediction analyses. The primary endpoint was pathological complete response (pCR). TOP2A and gene expression profiles were evaluated using pre-epirubicin biopsies. Gene expression data from ER-negative samples of the EORTC 10994/BIG 00-01 and MDACC 2003-0321 neoadjuvant trials were used for validation purposes.;	;	RESULTS: A pCR was obtained in 14% of the evaluable TOP patients. TOP2A amplification, but not protein overexpression, was significantly associated with pCR (p=0.001 and 0.22). We developed an “anthracycline-based score (A-Score)” that combines three signatures: a TOP2A gene signature and two previously published signatures related to tumor invasion and immune response. The A-Score was characterized by a high negative predictive value (NPV=0.98 [95% CI: 0.90-1.00]) overall, and in the HER2-negative and HER2-positive subpopulations. Its performance was independently confirmed in the anthracycline-based (FAC/FEC) arms of the two validation trials (BIG 00-01: 0.80 [0.61-0.92] and MDACC 2003-0321: 1.00 [0.80-1.00]).;	;	CONCLUSION: Given its high NPV, the A-Score could become, if further validated, a useful clinical tool to identify those patients who do not benefit from anthracyclines and could therefore be spared the non-negligible side effects.  |Predicting the efficacy of anthracyclines (epirubicin) in breast cancer (BC) patients (TOP trial);	;	120 microarray experiments from primary ER-negative breast tumors of anthracycline-treated patients.;	;	No replicate, no reference sample.                                                            |GPL570  |NA                        |   18260|       114|             1|
|GSE17705 |Endocrine Sensitivity Index Validation Dataset                                                                                             |[20697068](https://pubmed.ncbi.nlm.nih.gov/20697068/) |Validation dataset of 298 ER-positive patients treated with tamoxifen for 5 years.;	;	All patients in this dataset have ER+ breast cancer and were uniformly treated with tamoxifen for 5 years. The objective of the study was to correlate levels of genomic markers to outcomes (relapse free survival) in this cohort of uniformly treated patients.                                                          |Tissue samples were processed and profiled by two different labs (MD Anderson Cancer Center and Jules Bordet Institute).                                                        |GPL96   |JBI, MDACC |   12722| 103, 195|             1|
|GSE19615 |Integrated genomic and function characterization of the 8q22 gain                                                                          |[20098429](https://pubmed.ncbi.nlm.nih.gov/20098429/) |Integrated DNA and expression array analysis in primary human breast tumors identified chromosome 8q22 copy number gain and a suite of over-expressed genes in this region highly relevant to subsequent recurrence.;	8q copy gain is associated with increased risk of distant metastasis despite adjuvant chemotherapy and multiple genes from 8q22 are overexpressed and have pleiotropic effects, contributing to tumor growth, survival, and reduced killing of tumor cell by chemotherapy.                                                      |Affymetrix U133plus2 from 115  tumor patients                                                           |GPL570  |NA                        |   18260|       115|             1|
|GSE20181 |Letrozole (Femara) early and late responses to treatment                                                                                   |[20697427](https://pubmed.ncbi.nlm.nih.gov/20697427/) |In the present investigation, we have exploited the opportunity provided by neoadjuvant treatment of a group of postmenopausal women with large operable or locally advanced breast cancer (in which therapy is given with the primary tumour remaining within the breast) to take sequential biopsies of the same cancers before and after 10-14 days or 90 days treatment with letrozole.  RNA extracted from the biopsies has been subjected to Affymetrix microarray analysis and the data from paired biopsies interrogated to discover genes whose expression is most influenced by oestrogen deprivation.;	;	Keywords: Timecourse between subjects                                                           |Biopsies were taken from the same subjects at three timepoints: pretreatment, after 10-14 days Letrozol (2.5 mg/day, oral), and after 90 days Letrozol (2.5 mg/day, oral).                                                             |GPL96   |NA                        |   12722|        53|             1|
|GSE20194 |MAQC-II Project: human breast cancer (BR) data set                                                                                         |[20064235](https://pubmed.ncbi.nlm.nih.gov/20064235/) |The human breast cancer (BR) data set (endpoints D and E) was contributed by the University of Texas M. D. Anderson Cancer Center (MDACC, Houston, TX, USA). Gene expression data from 230 stage I-III breast cancers were generated from fine needle aspiration specimens of newly diagnosed breast cancers before any therapy. The biopsy specimens were collected sequentially during a prospective pharmacogenomic marker discovery study between 2000 and 2008. These specimens represent 70-90% pure neoplastic cells with minimal stromal contamination. Patients received 6 months of preoperative (neoadjuvant) chemotherapy including paclitaxel, 5-fluorouracil, cyclophosphamide and doxorubicin followed by surgical resection of the cancer. Response to preoperative chemotherapy was categorized as a pathological complete response (pCR = no residual invasive cancer in the breast or lymph nodes) or residual invasive cancer (RD), and used as endpoint D for prediction. Endpoint E is the clinical estrogen-receptor status as established by immunohistochemistry. RNA extraction and gene expression profiling were performed in multiple batches over time using Affymetrix U133A microarrays. Genomic analysis of a subset of this sequentially accrued patient population were reported previously. For each endpoint, the first 130 cases were used as a training set and the next 100 cases were used as an independent validation set.                                                               |Gene expression data from 230 stage I-III breast cancers                                                            |GPL96   |NA                        |   12722|       261|             1|
|GSE22226 |Heterogeneity of Response to Chemotherapy and Recurrence-free Survival in Neoadjuvant Breast Cancer: Results from the I-SPY 1 TRIAL        |[22198468](https://pubmed.ncbi.nlm.nih.gov/22198468/) |Introduction: The I-SPY 1 TRIAL was designed to evaluate complete pathologic response and tumor volume change, measured by magnetic resonance imaging (MRI), stratified by molecular subtypes and link response to 3-year recurrence free survival (RFS).Methods: Eligible patients had T =3 cm and received neoadjuvant chemotherapy with AC plus optional taxane.  Serial MRI, biopsy, and blood draws were conducted over the course of treatment, and a database of multiple molecular profiles was assembled.Results: A total of 221 patients were eligible for analysis: median tumor size was 6.0 cm and median age was 49 years.  The I-SPY 1 TRIAL patients had tumors with aggressive biology: 45% were estrogen receptor (ER) negative, 31% Her2+; and 91% high risk by the Netherlands Cancer Institute (NKI) 70-gene profile. After a median of 3.9 years, RFS was 77%.  HR/HER2 status improved predictability of RFS with the greatest difference between HR+/HER2- and triple-negative disease (hazard ratio 0.39).  Wound healing signature activation, p53 mutation, and Risk of Relapse (ROR) score were highly correlated and also significantly improved the accuracy of RFS predictions when added to stage. The rate of pathologic complete response (pCR) varied considerably from a low of 0-2% (NKI low risk, luminal A) to a high of 43-61% (17q Amplified, HER2 enriched, HR-/HER2+).  Both pCR and the more refined residual cancer burden (RCB) were significant predictors of RFS for all patients and even more predictive when analyzed within biomarker subsets.  Good risk (NKI low, ROR-S low risk, Wound Healing quiescent, p53 wild type) signatures were associated with significantly higher 3-year RFS than poor risk expression signatures (ROR-S high risk, Wound Healing Activated, p53 mutation, NKI high risk).Conclusion: Importantly, in this set of biologically poor prognosis tumors, pCR predicts for better outcome, especially when analyzed within breast cancer subsets.;	;	Keywords: reference x sample                                                             |reference x sample |GPL1708 |NA                        |   18507|       128|             2|
|GSE22358 |Clinical and Pathologic Response of the Primary Tumor in Women Receiving Neoadjuvant Docetaxel-Capecitabine (+/- Trastuzumab) Chemotherapy |[21373875](https://pubmed.ncbi.nlm.nih.gov/21373875/) |Results: In subjects who completed treatment and surgery, the pCR and near-complete response rates were 15.8% in HER2-negative and 50% in HER2-positive subjects. When stratified by genomic subtype, subjects of the HER2-enriched subtype had the best response (66.7%), the luminal A (11%) and B (4.8%) subtypes the poorest. Of 147 patients tested for p53 mutations using the AmpliChip test, 78 variants were detected; 55 were missense. The response rate among TP53 mutated patients was 30%, significantly higher than the rate in TP53 wild-type patients (10%, P = .0032). Concordance between AmpliChip mutation status versus IHC staining was 65%, with AmpliChip status being predictive of response and IHC status being not being predictive. Conclusion: Capecitabine plus docetaxel in HER2-negative subjects, and with trastuzumab in HER2-positive subjects, provided a good response rate with fewer cycles. p53 mutational analysis using the AmpliChip p53 assay, and genomic subtyping using the PAM50 assay, were promising predictive tests of response.;  |reference x sample      |GPL5325 |NA                        |   17220|       122|             2|
|GSE25055 |Discovery cohort for genomic predictor of response and survival following neoadjuvant taxane-anthracycline chemotherapy in breast cancer   |[21558518](https://pubmed.ncbi.nlm.nih.gov/21558518/) |PURPOSE: To develop a predictive test for response and survival following neoadjuvant taxane-anthracycline chemotherapy for HER2-negative invasive breast cancer.;	METHODS: We developed a microarray-based gene expression test from pre-treatment tumor biopsies (310 patients) to predict favorable outcome based on estrogen receptor (ER) status,pathologic response to chemotherapy, 3-year disease outcomes, and sensitivity to endocrine therapy. Tumors were classified as treatment-sensitive if predicted to have pathologic response (and not resistance) to chemotherapy, or sensitive to endocrine therapy. We tested predictive accuracy, with 95% confidence interval (CI), for pathologic response (PPV, positive predictive value), distant relapse-free survival (DRFS), and absolute risk reduction at median follow-up in 198 other patients. Independence from clinical-pathologic factors was assessed in a multivariate Cox regression analysis based on the likelihood ratio test. Other evaluable, published response predictors (genomic grade index (GGI), intrinsic subtype (PAM50), pCR predictor (DLDA30)) were compared.                                                          |Neoadjuvant study of 310 HER2-negative breast cancer cases treated with taxane-anthracycline chemotherapy pre-operatively and endocrine therapy if ER-positive.  Response was assessed at the end of neoadjuvant treatment and distant-relapse-free survival was followed for at least 3 years post-surgery.    |GPL96   |MDACC_M                   |   12722|       221|             1|
|GSE25065 |Validation cohort for genomic predictor of response and survival following neoadjuvant taxane-anthracycline chemotherapy in breast cancer  |[21558518](https://pubmed.ncbi.nlm.nih.gov/21558518/) |PURPOSE: To develop a predictive test for response and survival following neoadjuvant taxane-anthracycline chemotherapy for HER2-negative invasive breast cancer.;	METHODS: We developed a microarray-based gene expression test from pre-treatment tumor biopsies (310 patients) to predict favorable outcome based on estrogen receptor (ER) status,pathologic response to chemotherapy, 3-year disease outcomes, and sensitivity to endocrine therapy. Tumors were classified as treatment-sensitive if predicted to have pathologic response (and not resistance) to chemotherapy, or sensitive to endocrine therapy. We tested predictive accuracy, with 95% confidence interval (CI), for pathologic response (PPV, positive predictive value), distant relapse-free survival (DRFS), and absolute risk reduction at median follow-up in 198 other patients. Independence from clinical-pathologic factors was assessed in a multivariate Cox regression analysis based on the likelihood ratio test. Other evaluable, published response predictors (genomic grade index (GGI), intrinsic subtype (PAM50), pCR predictor (DLDA30)) were compared.                                             |Neoadjuvant validation cohort of 198 HER2-negative breast cancer cases treated with taxane-anthracycline chemotherapy pre-operatively and endocrine therapy if ER-positive.  Response was assessed at the end of neoadjuvant treatment and distant-relapse-free survival was followed for at least 3 years post-surgery.     |GPL96   |MDACC, USO                     |   12722|        71, 54|             1|
|GSE32646 |GSTP1 expression predicts poor pathological complete response to neoadjuvant chemotherapy in ER-negative breast cancer                     |[22320227](https://pubmed.ncbi.nlm.nih.gov/22320227/) |The purpose of the present study was to investigate the association of glutathione S-transferase P1 (GSTP1) expression with resistance to neoadjuvant paclitaxel followed by 5-fluorouracil/epirubicin/cyclophosphamide (P-FEC) in human breast cancers. The relationship of GSTP1 expression and GSTP1 promoter hypermethylation with intrinsic subtypes was also investigated. In this study, primary breast cancer patients (n = 123, stage II-III) treated with neoadjuvant P-FEC were analyzed. Tumor samples were obtained by vacuum-assisted core biopsy before P-FEC. GSTP1 expression was determined using immunohistochemistry, GSTP1 promoter methylation index (MI) using bisulfite methylation assay and intrinsic subtypes using DNA microarray. The pathological complete response (pCR) rate was significantly higher in GSTP1-negative tumors (80.0%) than GSTP1-positive tumors (30.6%) (P = 0.009) among estrogen receptor (ER)-negative tumors but not among ER-positive tumors (P = 0.267). Multivariate analysis showed that GSTP1 was the only predictive factor for pCR (P = 0.013) among ER-negative tumors. Luminal A, luminal B and HER2-enriched tumors showed a significantly lower GSTP1 positivity than basal-like tumors (P = 0.002, P < 0.001 and P = 0.009, respectively), while luminal A, luminal B and HER2-enriched tumors showed a higher GSTP1 MI than basal-like tumors (P = 0.076, P < 0.001 and P < 0.001, respectively). In conclusion, these results suggest the possibility that GSTP1 expression can predict pathological response to P-FEC in ER-negative tumors but not in ER-positive tumors. Additionally, GSTP1 promoter hypermethylation might be implicated more importantly in the pathogenesis of luminal A, luminal B and HER2-enriched tumors than basal-like tumors.      |Fresh frozen tumor samples obtained by vacuum-assisted core biopsy from one hundred and fifteen patients were subjected to RNA extraction and hybridization on Affymetrix microarrays.     |GPL570  |NA                        |   18260|       115|             1|
